### PR TITLE
[Smoke tests] Fix Expo clients installation

### PR DESCRIPTION
**Pull request by RedMickey**

### CODE REVIEW

File: test/smoke/package/src/helpers/setupEnvironmentHelper.ts [hunk 1]
```diff
@@ -158,61 +159,15 @@ export class SetupEnvironmentHelper {
...
-        });
-        installerProcess.on("error", (error) => {
-            console.log("Error occurred in expo-cli process: ", error);
-        });
-        await AndroidEmulatorHelper.checkIfAppIsInstalled(this.expoPackageName, 100 * 1000);
-        kill(installerProcess.pid, "SIGINT");
-        await sleep(1000);
+    // Installs Expo app on Android device using XDL function
+    public static async installExpoAppOnAndroid() {
+        await XDL.Android.installExpoAsync();
```
Comment (SounD120) [inline test/smoke/package/src/helpers/setupEnvironmentHelper.ts]: Please add logging here.

File: test/smoke/package/src/helpers/setupEnvironmentHelper.ts [hunk 2]
```diff
@@ -158,61 +159,15 @@ export class SetupEnvironmentHelper {
...
-                resolve();
-            });
-            installerProcess.on("error", (error) => {
-                console.log("Error occurred in expo-cli process: ", error);
-                reject(error);
-            });
-        });
+    // Installs Expo app on iOS device using XDL function
+    public static async installExpoAppOnIos() {
+        await XDL.Simulator._installExpoOnSimulatorAsync();
```
Comment (SounD120) [inline test/smoke/package/src/helpers/setupEnvironmentHelper.ts]: Please add logging here

Comment (RedMickey) [reply test/smoke/package/src/helpers/setupEnvironmentHelper.ts]: Added log string.

File: test/smoke/package/src/helpers/setupEnvironmentHelper.ts (hunk 1 above)
Comment (RedMickey) [reply test/smoke/package/src/helpers/setupEnvironmentHelper.ts]: Added log string.

**Pull request merged**